internal/rbd/controllerserver.go

```diff
@@ -1124,11 +1112,6 @@ func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi
 		return nil, err
 	}
 
-	if rbdVol.isEncrypted() {
```

Comment:
@nixpanic why few things related to encryption is removed in this PR?

Reply:
oh, that is not intentional, probably mixed up patches :man_facepalming: